pennylane_lightning/core/src/simulators/lightning_tensor/tensor/tncuda/TensorCuda.hpp

```diff
@@ -61,14 +63,16 @@ class TensorCuda final : public TensorBase<PrecisionT, TensorCuda<PrecisionT>> {
     /**
      * @brief Explicitly copy data from GPU device to host memory.
      *
-     * @param sv Complex data pointer to receive data from device.
+     * @param host_tensor Complex data pointer to receive data from device.
+     * @param length Number of elements to copy.
+     * @param async If true, the copy is asynchronous.
```

Comment:
Is `async=True` supported?

Reply:
Not really. It's a copy from the `DataBuffer` class